modules/nw-openstack-external-ccm.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// tbd
+// TODO: GitHub link clearance.
+
+:_content-type: CONCEPT
+[id="nw-openstack-external-ccm_{context}"]
+= The {rh-openstack} Cloud Controller Manager (CCM)
+
+In {product-title} 4.12, clusters that run on {rh-openstack-first} are switched from the legacy {rh-openstack} cloud provider to the external link:https://github.com/kubernetes/cloud-provider-openstack[OpenStack CCM]. This change follows the move in Kubernetes from in-tree, legacy cloud providers to external cloud providers that are implemented by using the link:https://kubernetes.io/docs/concepts/architecture/cloud-controller/[Cloud Controller Manager].
+
+To preserve user-defined configurations for the legacy cloud provider, existing configurations are mapped to new ones as part of the migration process. It searches for configurations in the following places:
+
+* `kube-cloud-config` in the `openshift-config-managed` namespace
```

Comment:
@stephenfin does CCM look for configurations in this config to transform them or only the one on openshift-config namespace?

Reply:
I checked in the code, for OpenStack it seems to using only `cloud-provider-config` in the `openshift-config` namespace
https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/pkg/controllers/cloud_config_sync_controller.go#L82-L123